guildwars2/guild/general.py

```diff
@@ -213,9 +247,46 @@ class GeneralGuild:
                         inline=False)
                     counter += 1
         if counter == 0:
-            return await ctx.send(
-                "No stash log entries yet for {}".format(guild_name.title()))
+            return await ctx.send("No stash log entries yet for {}".format(
+                guild_name.title()))
         try:
             await ctx.send(embed=data)
         except discord.Forbidden:
             await ctx.send("Need permission to embed links")
+
+    @guild.command(name="default", usage="<guild name>")
+    @commands.guild_only()
+    @commands.cooldown(1, 10, BucketType.user)
+    async def guild_default(self, ctx, *, guild_name=None):
+        """ Set your preferred guild for guild commands"""
+
```

Comment:
Help message could be expanded a bit, eg:
```py
""" Set your preferred guild for guild commands
Commands from the guild command group invoked 
without a guild name will default to this guild.

Invoke this command without an argument to reset the default guild.
"""
```

Reply:
Updated help message and added "for this Discord Server", so people know they can use different default guilds for different discord servers/guilds